pkg/smartcontract/zkpbinding/binding.go

```diff
@@ -112,7 +112,7 @@ var (
 func VerifyProof(a []byte, b []byte, c []byte, publicInput [][]byte) bool {
 	alphaPoint := crypto.Bls12381Deserialize(alpha)
 	betaPoint := crypto.Bls12381Deserialize(beta)
-	gammaPoint := crypto.Bls12381Deserialize(gamma)
+	gammaInversePoint := crypto.Bls12381Deserialize(gamma)
```

Comment:
Not related to integers. I see `examples/zkp/xor_compat/verify.go` is different, but which one is more correct? Should be a separate change anyway.